### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Joozty**
Waiting for https://github.com/oacore/design/pull/416

### CODE REVIEW

File: modules/search-layout/styles.module.css [hunk 1]
```diff
@@ -1,7 +1,9 @@
 .container {
   display: grid;
   grid-gap: 1rem;
-  grid-template-columns: calc(var(--search-results-percentage-width) * 100%) auto;
+  grid-template-columns: calc(var(--search-results-ratio-width) * 100%) calc(
```
Comment (Joozty) [inline modules/search-layout/styles.module.css:4]: main content 66% and sidebar 33%

**Comment by Joozty** [comment]
I feel like the /data-providers map is now too small :/

**Comment by viktor-yakubiv** [comment]
This alignment looks wrong to me. Any idea what could cause this?

services menu screenshot alignment is wrong

**Review by viktor-yakubiv — APPROVED** [review]
The layout looks good to me 👍

**Comment by Joozty** [comment]
You mean the menu drop-down doesn't match up with footer white area?

**Pull request merged**